### DISCUSSION TITLE
Web control can set config values to nonsense values

### DISCUSSION THREAD

**Discussion by worikgh** [Motion-Project/motion#1490] [General]
Raspberry Pi Zero
Built from Git/master

Using the web control interface I could set:
`locate_motion_mode` to  `purple` 

Reading the code at motion.c:2793, at first glance, this looks equivalent to it being set to `off`

**Comment by tosiara** [comment]
What would you like motion to do in this case?

**Comment by worikgh** [comment]
I would expect it not to change the value and return a HTTP 400 error

**Comment by tosiara** [comment]
Ok. And what should happen if you specify `locate_motion_mode purple` in your config? Should motion start and continue using default values or exit?

**Comment by worikgh** [comment]
I would expect startup to fail if there is nonsense in the configuration file.

**Comment by tosiara** [comment]
The idea was to force motion start and do the job regardless of any mistakes in the config. That's why, for example, motion starts anyway if the config does not exist at all

Will leave this as a feature request
